src/tests.rs

```diff
@@ -149,7 +149,7 @@ fn named_from_files_pass_tests() {
         let start_time = std::time::Instant::now();
         let crates = crates_data_from_file(&case);
         let mut dp = Index::new(&crates);
-        let root = new_bucket(&name, (&ver).into(), true);
```

Comment:
The corresponding change will need to be made in `named_from_files_pass_without_vers`.